.unimportedrc.json

```diff
@@ -0,0 +1,15 @@
+{
+  "rootDir": "./src",
+  "preset": "node",
+  "entry": ["src/**/*"],
```

Comment:
Adding every single source file as an entrypoint here effectively disables the tool altogether

Reply:
Ok will resolve this.